covid19-dashboard_NEW/main.py

```diff
@@ -0,0 +1,360 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from collections import defaultdict
+from datetime import datetime, timedelta
+
+from flask import Flask, send_from_directory
+from flask_caching import Cache
+from flask_cors import CORS
+from flask_compress import Compress
+
+from send_request import send_request
+from calculate_data import calculate_data
+
+config = dict(DEBUG=True,
+              CACHE_TYPE="filesystem",
+              CACHE_DIR='/tmp')
+
+app = Flask(__name__, static_folder='./build')
+Compress(app)
+
+CORS(app)
```

Comment:
Is CORS required? I think all requests are coming to this server, so let's avoid adding CORS if we can.

Reply:
Removed CORS.